### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

**Comment by Stebalien** [comment]
Build failure due to https://github.com/dgraph-io/badger/pull/1477.

File: ds_test.go [hunk 1]
```diff
@@ -546,7 +546,7 @@ func TestDiskUsage(t *testing.T) {
 		t.Fatal(err)
 	}
 	s, _ := d.DiskUsage()
-	if s <= 0 {
+	if s == 0 {
```
Comment (Stebalien) [inline ds_test.go:550]: Why did we need to change this?

Comment (gammazero) [reply ds_test.go:550]: @Stebalien I believe that was your change:
> fix lint error
https://github.com/ipfs/go-ds-badger2/pull/27/commits/1b16498077eff4df0079eb985e932d0a24aee342

Comment (Stebalien) [reply ds_test.go:550]: Ah... because disk usage is an unsigned int. Nevermind...

**Review by Stebalien — COMMENTED** [review]
LGTM (I submitted so I can't click the approve button)

**Comment by gammazero** [comment]
@aschmahmann If you could approve this (since neither stebalien nor I can)

**Review by willscott — APPROVED** [review]

**Pull request merged**